recipes/geocat-datafiles/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "geocat-datafiles" %}
+{% set version = "2022.03.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz # For building from a specific tag's tarball
+  sha256: 68aef597dbe4223fff37b11dab5e8720b1047085af58c95ddfa27d86c5fb75c9
+  # git_tag: v{{ version }} # For building from a specific tag
+  # git_rev: main # For building from branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed -vv .
+  # skip: true  # [py<38 or win]
+
+requirements:
+  build:
+    - python >=3.8                        # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - pooch                               # [build_platform != target_platform]
```

Comment:
```suggestion
```

I suggest you wait until you're in a feedstock to do this for cross-compiling; for now, it is not needed.

Reply:
Sure :)

Reply:
I want to add one thing for future reference: Since this is a noarch, this stuff is simply not needed. A noarch package can just be pulled into whatever arch without any modification (including pypy, arm64, etc.). But you may want to keep it for other packages which are noarch, up to you.

Reply:
Thanks.